pkgs/os-specific/linux/lvm2/default.nix

```diff
@@ -68,21 +68,18 @@ stdenv.mkDerivation rec {
   '';
 
 
-  patches = lib.optionals stdenv.hostPlatform.isMusl [
+  patches = let
+    version = "eae8e06eee6e93e59139cf4d06e8414f681e20c9";
+  in lib.optionals stdenv.hostPlatform.isMusl [
```

Comment:
Is there a chance we can apply those patches all the time?

Reply:
yes.